nservicebus/azure-service-bus/retries.md

```diff
@@ -5,36 +5,49 @@ tags:
+### Excessive prefetching and MaxDeliveryCount 
 
-In Azure Service Bus Versions 6 and below the default value for `MaxDeliveryCount` is 6. In Azure Service Bus Versions 7 and higher it has been changed to 10, based on the assumption that by default there are 2 instances of each endpoint and `MaxRetries` has the default value of 5.
+When messages are received, client receivers prefetch messages. Endpoint taking a long time to process a message will cause prefetched messages to lose lock tokens. The broker will consider that as a failed processing attempt increase `DeliveryCount` counter on the messages w/o processing attempt.  
 
+This can cause a message to be sent to the Delayed Retries or the error queue without being retried the desired number of times. The problem can be addressed by reducing the `PrefetchCount` on message receivers and retrying the failed messages.
 
-### Elastic Scale
+See [Message Receivers configuration](/nservicebus/azure-service-bus/configuration/full.md#controlling-connectivity-message-receivers) for more details.
 
-The `MaxDeliveryCount` setting becomes a property on the Azure Service Bus entity. It can't be easily modified after the entity has been created. At the same time the number of instances can be changed just by dragging a slider in the Azure administration portal.
 
-This becomes a problem when the number of instances exceeds what was originally planned. The messages can be dead lettered when the total retry value exceeds the value configured in `MaxDeliveryCount`, which was adjusted for a smaller number of instances.
+## Retries and dead letter queues
 
-One way to solve this problem is to configure the entity to forward dead letter messages to the error queue:
+A valid message failing processing will be retried and eventually end up in the error queue. If a message is poisonous or cannot be moved to an error queue, it will be moved by the broker to a dead letter queue. With multiple endpoints, monitoring multiple DLQs can be a challenge. One way to solve this problem is to configure the entity to forward dead letter messages to the error queue:
 
 snippet:forward-deadletter-conditional-queue
 
 
 ## Delayed Retries
 
-The problem with `MaxDeliveryCount` doesn't impact Delayed Retries. Since the message is deferred for a longer period of time, it will result in a resend operation inside the Azure Service Bus transport. That in turn will reset the delivery counter inside the brokered message.
+The problem with `MaxDeliveryCount` doesn't impact Delayed Retries. Since the message is deferred for a longer period of time, it will result in a resend operation inside the Azure Service Bus transport. That, in turn, will reset the delivery counter inside the brokered message.
```

Comment:
> The problem with `MaxDeliveryCount` doesn't impact Delayed Retries.

which problem? given that it's a new paragraph it's not obvious what we're talking about here. should we move? or state explicitly which problem we refer to?